Sources/View/ReusableView.swift

```diff
@@ -65,7 +80,8 @@ public extension UICollectionView {
 
     @available(iOS 9, *)
```

Comment:
I think we can probably remove this, as the project has had iOS 9 as deployment target for a while now 👴 (same on .podspec)